packages/training/python/src/training/ab_testing.py

```
@@ -0,0 +1,383 @@
+        start_time = time.time()
+        
+        try:
+            async with self._session.post(
+                f"{self.vllm_url}/chat/completions",
+                json={
+                    "model": model_name,
+                    "messages": messages,
+                    "max_tokens": 512,
+                    "temperature": 0.7,
+                },
+                timeout=aiohttp.ClientTimeout(total=60),
```

Ruff (0.14.10)

Undefined name `aiohttp`

(F821)